web/app/js/components/ConfigureProfilesMsg.jsx

```diff
@@ -53,21 +67,39 @@ class ConfigureProfilesMsg extends React.Component {
   };
 
   handleClose = () => {
-    this.setState({ open: false });
+    this.setState({
+      error: {
+        namespace: false,
+        service: false
+      },
+      open: false
+    });
   };
 
   handleChange = name => {
     let state = this.state;
 
     return e => {
       state.query[name] = e.target.value;
+      state.error[name] = false;
       this.setState(state);
     };
-  }
+  };
+
+  validateFields = (type, name) => {
+    let error = this.state.error;
+    let match = name ? (type === 'service' ?
```

Comment:
this double ternary makes it a little difficult to follow what's going on. I'd suggest something like:
```
  validateFields = (type, name) => {
    let error = this.state.error;

    if (_isEmpty(name)) {
      error[type] = false;
    } else {
      let match = type === 'service' ?
        serviceNameRegexp.test(name) :
        namespaceNameRegexp.test(name);

      error[type] = !match;
    }

    this.setState({ error });
  };
```